src/sql/parts/notebook/notebook.component.ts

```diff
@@ -153,6 +155,19 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 		}
 	}
 
+	public unselectActiveCell() {
+		if (!this._cellSelected) {
```

Comment:
Ah.... OK I'm guessing this logic is to handle the fact that unselectActiveCell _is_ called on clicking any cell? A quick search shows that you can pass the event to the click handler and then call stopPropagation on it to solve without need for tracking _cellSelected.
https://www.radzen.com/blog/angular-event-bubbling/
template change:

```html
(click)="selectCell($event, cell)
```

code change in selectCell
```ts
	public selectCell(event: Event, cell: ICellModel) {
        event.stopPropagation();
        ...
   }
```

Reply:
Ack, done